### PR TITLE
Reset the administrator's role

### PR DESCRIPTION
**Pull request by gergo85**
If I set a role for user, there is no way to delete from role. This solution add a "reset" feature.

### CODE REVIEW

**Comment by gergo85** [comment]
role

File: modules/backend/lang/en/lang.php [hunk 1]
```diff
@@ -121,6 +121,8 @@
         'email' => 'Email',
         'role_field' => 'Role',
         'role_comment' => 'Roles define user permissions, which can be overriden on the user level, on the Permissions tab.',
+        'role_none' => 'None',
+        'role_none_comment' => 'This administrator does not belong to any rules.',
```
Comment (LukeTowers) [inline modules/backend/lang/en/lang.php]: `any rules` should be `any roles`

File: modules/backend/models/User.php [hunk 2]
```diff
@@ -157,7 +157,7 @@ public function getGroupsOptions()
 
     public function getRoleOptions()
     {
-        $result = [];
```
Comment (LukeTowers) [inline modules/backend/models/User.php]: should be 
```php
$result = [
    0 => ['backend::lang.user.role_none', 'backend::lang.user.role_none_comment']
];
```
Don't access uninitialized variables, that's bad practice.

**Comment by gergo85** [comment]
I modified the code.

**Comment by LukeTowers** [comment]
Closing as it has been over a month since any activity on this occurred and we are trying to figure out what issues are still relevant. If this is still something that you would like to see through to fruition please respond and we can get the ball rolling.

**Pull request closed without merging**